FileOperationsCases.cs

```diff
@@ -0,0 +1,81 @@
+﻿using HelloWorld.Models;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json.Schema;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+
+namespace HelloWorld
+{
+     public class FileOperationsCases
+    {
+        public void WritingJsonToFile()
+        {
+            HomeActivity homeActivity = new HomeActivity("Ironing", 2, "Saturday");
+
+            string serialized = JsonConvert.SerializeObject(homeActivity, Formatting.Indented);
+
+            if (IsValidJson(serialized))
+                File.WriteAllText(@"C:\Users\obartkiv\source\repos\HelloWorld\FileA.txt", JsonConvert.SerializeObject(homeActivity));
+
+        }
+
+        public void ReadingFromFile()
+        {
+            List<HomeActivity> listOfActivities = new List<HomeActivity>();
+            string line;
+
+            StreamReader file = new StreamReader(@"C:\Users\obartkiv\source\repos\HelloWorld\FileB.txt");
+
+            while ((line = file.ReadLine()) != null)
+            {
+                if (line == string.Empty)
+                    continue;
+                string[] words = line.Split(',');
+                listOfActivities.Add(new HomeActivity(words[0], int.Parse(words[1]), words[2]));
```

Comment:
advice: try avoiding "magic numbers". e.g. you could add an enum with meaningful naming to store those indexes

Reply:
public enum ActivityData
{
Name = 0,
Duration =1,
Day = 3
}
HomeActivity(words[(int)ActivityData.Name], int.Parse(words[(int)ActivityData.Duration]), words[(int)ActivityData.ActivityData])